src/index.html

```diff
@@ -6,14 +6,406 @@
       name="viewport"
       content="width=device-width, initial-scale=1.0"
     />
-    <title>Title</title>
+    <title>Eco</title>
     <link
       rel="stylesheet"
       href="./styles/main.scss"
     />
+    <link
+      rel="shortcut icon"
+      href="logo.svg"
+    />
   </head>
   <body>
-    <h1>Hello Mate Academy</h1>
+    <div class="wrapper">
+      <header class="header">
+        <div class="header__container">
+          <a href="#" class="header__logo icon">
+            <img src="images/logo.svg" alt="logo" />
+    
+            <span>witch of mountain winds</span>
+          </a>
+    
+          <div class="header__menu menu">
+            <a href="#menu" class="menu__open icon icon--open"></a>
+    
+            <nav id="menu" class="menu__body">
+              <a href="#" class="menu__close icon icon--close"></a>
+    
+              <ul class="menu__list">
+                <li class="menu__item">
+                  <a href="#shop" class="menu__link">Shop</a>
+                </li>
+    
+                <li class="menu__item">
+                  <a href="#contact" class="menu__link">Contact</a>
```

Comment:
This links to `#contact` but your contact block has `id="contacts"`, ending with `s`.